x-pack/plugins/timelines/public/components/t_grid/standalone/index.tsx

```diff
@@ -36,17 +36,18 @@ import type { State } from '../../../store/t_grid';
 import { useTimelineEvents } from '../../../container';
 import { StatefulBody } from '../body';
 import { LastUpdatedAt } from '../..';
-import {
-  SELECTOR_TIMELINE_GLOBAL_CONTAINER,
-  UpdatedFlexItem,
-  UpdatedFlexGroup,
-  FullWidthFlexGroup,
-} from '../styles';
+import { SELECTOR_TIMELINE_GLOBAL_CONTAINER, UpdatedFlexItem, UpdatedFlexGroup } from '../styles';
 import { InspectButton, InspectButtonContainer } from '../../inspect';
 import { useFetchIndex } from '../../../container/source';
 import { AddToCaseAction } from '../../actions/timeline/cases/add_to_case_action';
 import { TGridLoading, TGridEmpty, TimelineContext } from '../shared';
 
+const FullWidthFlexGroup = styled(EuiFlexGroup)<{ $visible: boolean }>`
```

Comment:
@XavierM `FullWidthFlexGroup` has a hardcoded `min-height`, so I stopped using the one defined in the styles and redefined a new one. Do we actually need this hardcoded min-height? Even integrated tGrid gets rid of the min-height https://github.com/elastic/kibana/blob/main/x-pack/plugins/timelines/public/components/t_grid/integrated/index.tsx#L83
<img width="1166" alt="Screenshot 2022-02-11 at 11 43 29" src="https://user-images.githubusercontent.com/2852703/153578321-cdeaef4c-93d8-4b27-a98e-f5e144b6029a.png">

Reply:
@mgiota either way should be fine 👍

Reply:
@afgomez It looks like `min-height` is not being used at all in the `integrated` version of `t-grid` and with this PR is it also removed from the `standalone` version of t-grid. Soon Alerts 2.0 is coming, where quite many things will be cleaned up. I suggest we keep it as it is for now if you don't mind.